test/controllers/contests_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ContestsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Can you explain what this file is?

Reply:
Ah. Ian used rails to generate the controller, but that does automatically create a test file. We will remove.